src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentProfile.tag

```diff
@@ -23,7 +23,11 @@
             <tr>
               <td class="text-bold">Short Name (Gender)</td>
               <td>${empty student.shortName ? none : fn:escapeXml(student.shortName)}
-                (<i> ${student.gender == other ? noneForGender : fn:escapeXml(student.gender)} </i>)
+                (<i>
+                   <span class="text-muted">
```

Comment:
This is not accurate because now this class gets added regardless of the gender. It should only be added if the gender is "Not specified".

Reply:
Sir @wkurniawan07 I am not able to understand you what exactly I have to do here

Reply:
As here I as the behaviour required by the page I have to show student short name and it's gender adjecent to short name in the bracket so I am using student.displayedGenderType to show the the gender .
Sir Kindly guide me Related to the change you wish to see I will be thankful to you